cumulusci/robotframework/utils.py

```diff
@@ -199,3 +200,34 @@ def run_with_retry(self, *args, **kwargs):
     else:
         # Decorator was used without arguments
         return decorate(target)
+
+
+def perfJSON2Dict(jsondata, include_raw):
```

Comment:
Strictly speaking there is no reason to instantiate an object because there is no state, but sure I could wrap them in an object for a namespace...

Reply:
f468c120ca1d

Reply:
The state would be the actual performance data, wouldn't it? Instead of having a `jsondata` object that you pass in, the instance would be the `jsondata` object. In other words, instead of `perfJSON2Dict(jsondata, True)` you would do `jsondata.to_dict(True)`.

Reply:
0aba6230f511ce49a0d7e0d67208c1d73f1dc63f